test_unstructured_ingest/check-diff-evaluation-metrics.sh

```diff
@@ -13,43 +13,50 @@ set -e
 SCRIPT_DIR=$(dirname "$(realpath "$0")")
 OVERWRITE_FIXTURES=${OVERWRITE_FIXTURES:-false}
 TMP_DIRECTORY_CLEANUP=${TMP_DIRECTORY_CLEANUP:-true}
+OUTPUT_FOLDER_NAME=$1
 OUTPUT_ROOT=${OUTPUT_ROOT:-$SCRIPT_DIR}
-TMP_METRICS_LATEST_RUN_DIR=$OUTPUT_ROOT/metrics-tmp
-METRICS_DIR=$OUTPUT_ROOT/metrics
+TMP_METRICS_LATEST_RUN_DIR=$OUTPUT_ROOT/metrics-tmp/$OUTPUT_FOLDER_NAME
+METRICS_DIR=$OUTPUT_ROOT/metrics/$OUTPUT_FOLDER_NAME
```

Comment:
for clarity, add a comment above of what this could be, e.g.:

# e.g. METRICS_DIR could be test_unstructured_ingest/metrics/element-type or test_unstructured_ingest/metrics/text-extraction